migrate_stakeholders_dag.py

```diff
@@ -52,9 +52,12 @@
 
     postgresConnId = "postgres_default"
     db = PostgresHook(postgres_conn_id=postgresConnId)
-    conn = db.get_uri()  
+    conn = db.get_uri()
+    # Airflow Variable DATABASE_PASSWORD is masked in the Airflow Variables UI and logs but not when actually used
+    # https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/security/secrets/mask-sensitive-values.html
+    # tested working in the https://github.com/Greenstand/treetracker-airflow-dags/blob/main/test1.py DAG
     environments = {
-        'DATABASE_URL': Variable.get("DATABASE_URL"),
+        'DATABASE_URL': "postgresql://" + Variable.get("DATABASE_LOGIN") + ":" + Variable.get("DATABASE_PASSWORD") + "@" + Variable.get("DATABASE"),
```

Comment:
@huang-pan the problem I knew is that when you trying to get the DATABASE_PASSWORD, it get masked by airflow, the this connection string is wrong

Reply:
No it's not, did you not see my thorough explanation in https://github.com/Greenstand/treetracker-airflow-dags/issues/156 and https://github.com/Greenstand/treetracker-airflow-dags/issues/103?

It's only masked in the Airflow UI and logs. It's not masked when you pass the variable to the function that's using it. I know this for a fact because we use this method at Roofstock and it works.